slsim/Sources/source.py

```diff
@@ -173,3 +233,78 @@ def point_source_position(self, center_lens, draw_area):
             )
             return self._center_point_source
         return extended_source_center
+
+    def kwargs_extended_source_light(
+        self, center_lens, draw_area, band=None, sersic_profile_str="single"
+    ):
+        """Provids dictionary of keywords for the source light model(s).
+
+        :param band: Imaging band
+        :param sersic_profile_str: number of sersic_profile
+        :type sersic_profile_str: str . eg: "single" or "double".
+        :return: dictionary of keywords for the source light model(s)
+        """
+        if band is None:
+            mag_source = 1
+        else:
+            mag_source = self.extended_source_magnitude(band=band)
+        center_source = self.extended_source_position(
+            center_lens=center_lens, draw_area=draw_area
+        )
+        if sersic_profile_str == "single":
+            size_source_arcsec = float(self.angular_size) / constants.arcsec
+            kwargs_extended_source = [
+                {
+                    "magnitude": mag_source,
+                    "R_sersic": size_source_arcsec,
+                    "n_sersic": float(self.n_sersic),
+                    "e1": float(self.ellipticity[0]),
+                    "e2": float(self.ellipticity[1]),
+                    "center_x": center_source[0],
+                    "center_y": center_source[1],
+                }
+            ]
+        elif sersic_profile_str == "double":
+            # w0 and w1 are the weight of the n=1 and n=4 sersic component.
+            if (
+                "w0" not in self.source_dict.colnames
+                or "w1" not in self.source_dict.colnames
+            ):
+                w0 = 0.5
```

Comment:
does this imply that the light is split 50/50 or what are these variables? I would suggest that you demand full knowledge of these things from the dictionary and not make arbitrary choices in this definition about it